python-client/giskard/models/utils.py

```diff
@@ -10,3 +12,12 @@ def map_to_tuples(data: Iterator):
         return data
 
     return map(lambda x: (x,), data)
+
+
+def numpy_encoder(some_list: List):
+    if some_list is not None:
+        if isinstance(some_list[0], np.integer):
+            return [int(i) for i in some_list]
```

Comment:
there's a [.item()](https://numpy.org/doc/stable/reference/generated/numpy.ndarray.item.html) method that can do it for numerous numpy types that we can reuse

Reply:
also since we're iterating over the whole array anyway it's safer to do the check on each element:

```
def numpy_encoder(some_list: List):
    if some_list is None:
        return None
    else:
        return [i.item() if isinstance(i, np.generic) else i for i in some_list]
```

Reply:
I didn't know about this one, thanks!